javascript/code-challenges/challenges-02.test.js

```diff
@@ -0,0 +1,343 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+
+CHALLENGE 1 - Review
+
+Write a function named raisedToTheThird that takes in an array of numbers and returns a new array of each of those numbers raised to the 3rd power (hint: look up Math.pow()). Use forEach to solve this problem.
+
+------------------------------------------------------------------------------------------------ */
+
+function raisedToTheThird(numbers) {
+  let result = [];
+  numbers.forEach(function(number) {
+    var power = Math.pow(number, 3);
```

Comment:
Var is a bit outdated, we want to stick to let or const when we declare variables!